### ISSUE TITLE
Size to fit in repo file browser truncates text

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#171]
```
What steps will reproduce the problem?
1. Open repository (for example, http://svnx.googlecode.com/svn/)
2. In file browser double click on separator between [branches, tags, trunk, 
wiki] and empty pane.
3. Observe the second pane (with [branches, tags, trunk, wiki])

What is the expected output? What do you see instead?
I expect second pane to be resized so all the text is visible, but not wider, 
i.e. size to fit. Actual result is that 'branches' is truncated, i.e. I see 
[bra…, tags, trunk, wiki].

What version of the product are you using? On what operating system?
svnX 1.3.3 on Mac OS X 10.6.8.
```

Original issue reported on code.google.com by `vsap...@gmail.com` on 10 Jun 2012 at 1:25

**Comment by GoogleCodeExporter** [comment]
```
I was unaware of this double-click behaviour.
I’ve now added the necessary code to support it correctly.
[This will appear in a future release of svnX.]
Thanks for your report.
```

Original comment by `chris...@gmail.com` on 10 Jun 2012 at 5:15
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svnX 1.3.4.
```

Original comment by `chris...@gmail.com` on 4 Sep 2012 at 5:35
- Changed state: **Fixed**
